committees/project-structure.md

```diff
@@ -0,0 +1,57 @@
+# Rust Project Structure Committee
+
+## Mission
+
+The *Rust Project Structure* committee is tasked with determining the "shape" of the Rust Project and its membership,
+and to make proposals to the Council on any recommended changes,
+and to produce documentation on clarifying the Project structure.
```

Comment:
I think this would be a bit clearer as a list:

```suggestion
The *Rust Project Structure* committee is tasked with:

- Determining the "shape" of the Rust Project and its membership
- Making proposals to the Council on any recommended changes
- Producing documentation clarifying the Project structure
```